build-tools/packages/build-tools/tsconfig.json

```diff
@@ -15,6 +15,8 @@
         "types": [
             "node"
         ],
+        "esModuleInterop": true,
```

Comment:
we can move `data/exclusions.json` to the build-cli and remove this from tsconfig. Later import exclusions.json the same way --

`const exclusions: RegExp[] = require('../../data/exclusions.json').map((e: string) => new RegExp(e, "i"));`